SSL-Index/app.R

```diff
@@ -428,6 +446,31 @@ server <- function(input, output, session) {
       # tabItem("teamindex",teamIndexUI(id = "teamindex")),
     )
   })
+
+
+  #### User FAB ###
+
+  # Hides the user FAB if visitor isn't logged in
+  observe({
+    if (any(c(0,5) %in% authOutput()$usergroup)) {
+      removeUI(
+        selector = ".homemade-user-fab"
+      )
+    }
+  }) %>%
+    bindEvent(authOutput())
+
+  observeEvent(input$player, {
```

Comment:
OK they go here. We should probably look into combining observers in some way as this would increase the number of observers needed.

Reply:
Sounds reasonable. But I'm not familiar with how to combine observers, do we have any examples of this in the code?

Reply:
bindEvent() can handle multiple inputs. Ex.
```
bindEvent(
  combineTriggers(input$resetUpdate, input$resetRegression),
  ignoreInit = TRUE
)
```
in playerUpdateBox. I am not sure if `combineTriggers()` is actually needed as it should just support a list of inputs directly.

The second issue would be if we can actually tie it into the input$tabs observer which is the one that is currently changing the page. With the current way of navigating it does not so we'd need a separate observer.

Reply:
That's good to know. But to your second issue - I'm not sure how to connect the user FAB events to reuse the`input$tabs` observer. I think they might need to be separate for now, as the FAB observer updates the selected tab which in turn triggers the `input$tabs` observer to change the page. If I've understood the flow of things correctly.